src/fcntl.rs

```diff
@@ -380,3 +380,15 @@ pub fn fallocate(fd: RawFd, mode: FallocateFlags, offset: libc::off_t, len: libc
     let res = unsafe { libc::fallocate(fd, mode.bits(), offset, len) };
     Errno::result(res)
 }
+
+/// Allocate file space.
+///
+/// Ensure disk space is allocated for the file referred to by file
+/// descriptor fd for the bytes in the range starting at offset and
+/// continuing for len bytes.
+/// http://pubs.opengroup.org/onlinepubs/9699919799/
```

Comment:
The link is wrong.  I think you probably linked to the wrong iframe.  Also, for consistency with other Nix functions, you should format the link like this:
```rust
/// # References
///
/// [`posix_fallocate`](http://whatever)
```